### ISSUE TITLE
Update Stack Overflow link on client library ref landing page for Python

### ISSUE THREAD

**Issue by omaray** [GoogleCloudPlatform/google-cloud-python#2421]
On the [current landing page for google-cloud-python](http://googlecloudplatform.github.io/google-cloud-python) there is a Stack Overflow icon:

![image](https://cloud.githubusercontent.com/assets/6609430/18824123/8f7ada86-8374-11e6-9807-687667b0d488.png)

[Current Result]
That icon currently points to http://stackoverflow.com/questions/tagged/google-cloud-python (which was a good attempt!)

[Expected Result]
The icon should point to: http://stackoverflow.com/questions/tagged/google-cloud-platform+python

**Comment by dhermes** [comment]
@omaray This is great in theory but user behavior in the wild may also be relevant.

1. http://stackoverflow.com/questions/tagged/gcloud-python has 56 questions that need to be retagged
1. Sometimes people say `gcloud` when they mean our library: http://stackoverflow.com/questions/tagged/gcloud+python
1. Alternate tags like http://stackoverflow.com/questions/tagged/python-2.7 have non-triviial traffic

**Comment by omaray** [comment]
Hey - yes, here are some answers.

1. I'm not too concerned with previous questions and tags - 56 is not a horrible number. But also, more importantly, anyone going to stay Pub/Sub and having a question about say "Python library" will be directed to the [python]+[google-cloud-pubsub] multi-tag search! And many of the results you point me too already have that in place (meaning there are multiple tags including the [python] tag and the [product] tag).

In other words we are doing primarily [lang]+[tech] multi-tag searches instead of having one tag per client lib.

2. The second point you mention is exactly why we renamed actually from gcloud to google-cloud. 

3. Yeah, that I think we need to live with. We will focus on the actual language tag. If people want to add a precision in terms of version that's fine - but there is little we can here and we can cover it all.

Let me know if that makes sense.

**Comment by dhermes** [comment]
@omaray I'm going through and retagging right now.

http://stackoverflow.com/questions/tagged/gcloud-python has a logo and some info because it's a sponsored tag

http://stackoverflow.com/questions/tagged/google-cloud-python has nothing right now

**Comment by omaray** [comment]
Hey Danny - just to clarify:

(1) what do you mean by retagging exactly? So you remove the old one?

(2) just to clarify, for now let's go with  http://stackoverflow.com/questions/tagged/google-cloud-platform+python as the solution; and I'm waiting on some internal answers (which might take a bit of time). So let's stay consistent across all languages and use this new approach - and worst case we'll update again later on (should be an easy fix).

**Comment by dhermes** [comment]
> what do you mean by retagging exactly? So you remove the old one?

In most cases I just added the new one, so there are lots of questions with both `google-cloud-python` and `gcloud-python`. StackOverflow has a max of 5 though so in that case I deleted `gcloud-python` if it made sense (more often in the 5 tag case, the question was about the `gcloud` CLI, not our library, so it was not really relevant).

----

As for going with `google-cloud-platform` as the tag, it's your call but there are a few issues I see on first impression:

- We can't **make** people use that combo. People will start typing the name of the library in the tag UI and wait until they see it autocomplete. When they post a question on SO, they (with 99% certainty) won't have any idea what our blessed way to tag is.
- `google-cloud-platform` + `python` will be much broader than this library, e.g. most of the `gcloud` CLI is written in Python, GAE has a Python runtime, many other teams use Python data libraries like `pandas`

**Comment by omaray** [comment]
Yes, I'm aware of these limitations. And other limitations too. For now let's stick with that (cause in theory it does't hurt per se). And I'll keep following up with the Cloud Support team to ensure this is the right strategy. And I am waiting on some answers back for how SO might or might not support anonymous searches through tags.

**Comment by lukesneeringer** [comment]
Hello,
One of the challenges of maintaining a large open source project is that sometimes, you can bite off more than you can chew. As the lead maintainer of `google-cloud-python`, I can definitely say that I have let the issues here pile up.

As part of trying to get things under control (as well as to empower us to provide better customer service in the future), I am declaring a "bankruptcy" of sorts on many of the old issues, especially those likely to have been addressed or made obsolete by more recent updates.

My goal is to close stale issues whose relevance or solution is no longer immediately evident, and which appear to be of lower importance. I believe in good faith that this is one of those issues, but I am scanning quickly and may occasionally be wrong. If this is an issue of high importance, please comment here and we will reconsider. If this is an issue whose solution is trivial, please consider providing a pull request.

Thank you!
